source/2019-09-27-the-ember-times-issue-117.md

```diff
@@ -21,13 +22,13 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [A newly adopted Ember Addon 🐣](https://twitter.com/Alonski/status/1177577454711582720)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Adopted Ember Addons](https://github.com/adopted-ember-addons) has adopted [ember-sortable](https://github.com/adopted-ember-addons/ember-sortable), an addon which provides Sortable UI primitives for Ember. The addon was created by [Jamie White](https://github.com/jgwhite) over four years again and he has been maintaining it all this time! The time has come however for the addon to get some new love ❤ under the Adopted Ember Addons umbrella. 
```

Comment:
Alon, in addition to what Ilya suggested, can I recommend adding Jamie's GitHub handle next to his name? `[Jamie White (@jgwhite)][https://github.com/jgwhite]`